openedx/core/lib/block_structure/factory.py

```diff
@@ -82,12 +81,9 @@ def create_from_cache(cls, root_block_usage_key, block_structure_cache):
 
         Raises:
             BlockStructureNotFound - If the root_block_usage_key is not found
-                in the cache.
+                in the store.
         """
-        block_structure = block_structure_cache.get(root_block_usage_key)
-        if block_structure is None:
-            raise BlockStructureNotFound('Block structure for {} not found in the cache.'.format(root_block_usage_key))
-        return block_structure
+        return block_structure_store.get(root_block_usage_key)
```

Comment:
**Note to reviewers:** Exception raised from within the `get` call so no need to do so here.